src/attributes.jl

```diff
@@ -349,6 +350,7 @@ struct ListOfVariableAttributesSet <: AbstractInstanceAttribute end
 A string identifying the variable. It is invalid for two variables to have the same name.
 """
 struct VariableName <: AbstractVariableAttribute end
+mustcopy(::VariableName) = false
```

Comment:
Eh, why is this a special case? Solvers should generally support names.

Reply:
I just thought that instance used through an instance manager do not have to support it. But it's easy to implement in the wrapper so we might as well do it.